sidecar/sidecar.go

```diff
@@ -110,8 +110,7 @@ func Run(conf config.Config) error {
 			return err
 		}
 
-		// TODO: Log failure?
-		go appSupervisor.DoLogManagement("/tmp/filebeat.yml")
+		appSupervisor.AddProcess([]string{"filebeat", "-c", "/tmp/filebeat.yml"}, []string{"GODEBUG=netdns=go"})
```

Comment:
Yes, we would get rid of the -log and -logstash_host flags completely. From our perspective, wouldn't care what the helper process is. The only semantics we offer is that termination of a helper process does not result in termination of the sidecar itself. But termination of the primary process will result in termination of the sidecar.

```yaml
supervise:
  app: python /opt/microservices/productpage.py 9080 http://localhost:6379
  helpers:
     - filebeat -c /etc/filebeat.yaml
     - statsd -x -y -z -a -b
     - logrotate -a -b -c
```